app/templates/components/claim-user.hbs

```diff
@@ -0,0 +1,12 @@
+<span {{action 'togglePopup'}}>{{author.unregisteredContributor}}
+    {{#bs-tooltip}}{{t 'components.claim-user.tooltip_message'}}{{/bs-tooltip}}
+    {{#bs-popover placement='bottom' title=(t 'components.claim-user.title') visible=showPopup}}
+        {{#click-outside action=(action 'hidePopup') }}
+            <div style="display: flex">
```

Comment:
Normally it could either be an inline style or a css class. Be since we don't use css files that's local to only the component in `ember-osf-preprints` (we do use local css files for `ember-osf`), it means I have to put it either in `app.scss` or other similar files in the `app/styles` directory, which is something I want to avoid as much as possible.